.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# Why
```

Comment:
may we add a "Resolves #" section to top?

Reply:
I made some changes, with a note about using github's keywords for this